Raymond-Osy/Politico
# #163387853 Initial server setup

#### What does this PR do?
This is the initial server setup for this application

#### Description of Task to be completed?
- Add a package.json file
- Add the express app
- Add babel for transpiling Es6 syntax down
- Add Eslint and use the airbnb style guide for linting
- Add the .gitignore file for git to ignore large files

#### How should this be manually tested?
- Do a git clone in your local computer
- Open the console/comandline
- type npm run dev
- Open a browser and visit http://localhost:3000/ then you should see a response to show that the server is up and running

#### Any background context you want to provide?
N/A

#### What are the relevant pivotal tracker stories?
[#163387853](https://www.pivotaltracker.com/n/projects/2239146)

#### Screenshots (if appropriate)
N/A

#### Questions:
N/A
